src/Command/Anonymization/AnonymizeCommand.php

```diff
@@ -247,31 +248,20 @@ private function doBackupInitialDatabase(): void
         }
 
         $backupper = $this->backupperFactory->create($this->connectionName);
-
-        $this->initialDatabaseBackupFilename = $this->storage->generateFilename($this->connectionName, $backupper->getExtension());
+        $this->initialBackupFilename = $this->storage->generateFilename($this->connectionName, $backupper->getExtension());
 
         $backupper
-            ->setDestination($this->initialDatabaseBackupFilename)
+            ->setDestination($this->initialBackupFilename)
+            ->addLogger(new ConsoleLogger($this->io))
             ->setVerbose($this->io->isVerbose())
```

Comment:
You should remove the setVerbose stuff, I really do think it's not relevant anymore.

Reply:
As I said previously in the related discussion:

> In fact, setVerbose() is not initially used to control the logging verbosity of the backupper itself, it is mainly to tell the backupper to call the binary in verbose mode. I will keep it, but I will add the logger independently of the verbosity level, ConsoleLogger handles verbosity by itself.